swing/src/main/java/info/openrocket/swing/gui/main/SimulationPanel.java

```diff
@@ -1456,6 +1457,21 @@ public int getDefaultWidth() {
 						}
 					},
 
+					//// Simulation RK Solver
+					// TODO: add translation
+					new Column("RK Method") {
```

Comment:
Again, RK method is not generic enough

Reply:
Should be all set in `f4d47579d68af4c9350bacf9f3e73703ea254a91`.